### PR TITLE
CI: xfail failing parquet test

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
ok by me

**Comment by jreback** [comment]
is there an issue ref for this?

**Comment by jbrockmendel** [comment]
> is there an issue ref for this?

no

File: pandas/tests/io/test_parquet.py [hunk 1]
```diff
@@ -564,8 +565,19 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa, s3so):
...
-    @td.skip_if_no("s3fs")
-    @pytest.mark.parametrize("partition_col", [["A"], []])
+    @td.skip_if_no("s3fs")  # also requires flask
+    @pytest.mark.parametrize(
+        "partition_col",
+        [
+            pytest.param(
+                ["A"],
+                marks=pytest.mark.xfail(
+                    PY38, reason="Getting back empty DataFrame", raises=AssertionError,
```
Comment (jreback) [inline pandas/tests/io/test_parquet.py:575]: right is this something we are controlling on pandas side? or pyarrow / fastparquet?

Comment (jbrockmendel) [reply pandas/tests/io/test_parquet.py:575]: No idea cc @jorisvandenbossche ?

Comment (jreback) [reply pandas/tests/io/test_parquet.py:575]: ahh i c this is failing PRs ok let'e merge and can always come back to this (@jbrockmendel if you can make an issue about this)

**Comment by simonjayhawkins** [comment]
@meeseeksdev backport 1.1.x

**Comment by simonjayhawkins** [comment]
@WillAyd changing the milestone and not backporting. this test is not failing on 1.1.x

**Comment by WillAyd** [comment]
@jbrockmendel OK by you?

**Comment by jbrockmendel** [comment]
yep

**Comment by TomAugspurger** [comment]
Do we have an issue to track this? Could anyone debug if it just the test setup (moto or something?) that was causing us to return empty dataframes, or do we have an actual issue?

**Pull request merged**